anaconda_project/internal/cli/main.py

```diff
@@ -165,6 +167,31 @@ def add_env_spec_name_arg(preset, required):
     preset.add_argument('-u', '--user', metavar='USERNAME', help='User account, defaults to the current user')
     preset.set_defaults(main=download.main)
 
+    preset = subparsers.add_parser('dockerize', help="Build a docker image of the Anaconda Project.")
+    add_directory_arg(preset)
+    preset.add_argument('-t',
+                        '--tag',
+                        default=None,
+                        help='Tag of the output docker image in the format name:tag. '
+                        'Default: "<project-name>:latest", where <project-name> is taken from '
+                        'the name tag in the anaconda-project.yml file.')
+    preset.add_argument(
+        '--command',
+        default='default',
+        help='Select the command to run. If unspecified the "default" command is run.\nThe default command '
+        'is defined as either the command named "default" or the first command specified in the '
+        'anaconda-project.yml file.')
```

Comment:
```suggestion
        'is defined as either the command named "default" (if any) or (otherwise)  '
        'the first command specified in the anaconda-project.yml file.')
```